src/patito/pydantic.py

```diff
@@ -50,6 +52,23 @@
 # Should be a typed subclass of Model.
 ModelType = TypeVar("ModelType", bound="Model")
 
+PT_INFO = Literal["constraints", "derived_from", "dtype", "unique"]
+
+_Unset: Any = PydanticUndefinedAnnotation
+
+
+class PatitoFieldInfo(fields.FieldInfo):
```

Comment:
The warning can be further communicated by the class name:

```suggestion
class _PatitoFieldInfo(fields.FieldInfo):
```

Reply:
Thanks, addressed by https://github.com/JakobGM/patito/pull/32/commits/386ba8385ec07c3db464010068aa1cb2933fcec9